src/pages/jiyeon/Login/LoginJiyeon.js

```diff
@@ -1,7 +1,61 @@
 import React from 'react';
+import './LoginJiyeon.scss';
+import { Link } from 'react-router-dom';
+import '../../../styles/jiyeon/mixin.scss';
 
-function LoginJiyeon() {
-  return <h1>hi</h1>;
-}
+import { useState } from 'react';
+
+const LoginJiyeon = () => {
+  // const [value, setValue] = useState();
+  const [id, setUserId] = useState('');
+  const [pw, setUserPw] = useState('');
+  const [loginBtnstate, setLoginBtnstate] = useState(false);
+
+  const saveUserId = e => {
+    setUserId(e.target.value);
+  };
+
+  const saveUserPw = e => {
+    setUserPw(e.target.value);
+  };
+
+  const loginBtn = () => {
+    //if (id.indexOf(@) && pw.length > 4) =>true
+    return id.includes('@') && pw.length > 4 ? true : false;
+  };
+
+  return (
+    <>
+      <div id="wrapLogin">
+        <h1>Westagram</h1>
+        <input
+          id="id"
+          type="text"
+          placeholder="전화번호, 사용자 이름 또는 이메일"
+          defaultValue={id}
+          onChange={saveUserId}
+        />
+        <input
+          id="password"
+          type="password"
+          placeholder="비밀번호"
+          defaultValue={pw}
+          onChange={saveUserPw}
+        />
+        <Link to="../MainJiyeon">
```

Comment:
```suggestion
        <Link to="/MainJiyeon">
```
to 속성에는 해당 파일의 경로가 아닌 Router 컴포넌트에서 설정해 주었던 도메인의 주소가 들어가야 합니다!